### DISCUSSION TITLE
Meeting displays correctly on our website with plugin but is not picked up by meeting guide app.

### DISCUSSION THREAD

**Discussion by dev4a8** [code4recovery/12-step-meeting-list#1408] [Q&A]
our web site, https://ncsandiegoaa.org/, covers North San Diego County including Marine Corps Base Camp Pendleton.  We have 6 meetings aboard Camp Pendleton and we can get all 6 to display correctly, including excellent directions to the meetings.  The meeting Guide App only picks up 4 of these meetings.  Camp Pendleton is problematic in that for the most part there are no street addresses.  Locations on base are obtained by building numbers, street intersections, etc.. If street intersections are unique enough, Santa Margarita Rd & Roosevelt Dr, Camp Pendleton North for example, this will sometimes work.  A more generic address like 3rd & B St. for example will show up in another city 40 miles away.  Using a Google Plus Code, CCFV+CG or 9GG9+82 for the meeting that we are having trouble with will work on our site but the meetings don't get added to the app. I have tried many different ways of trying to enter the location, for instance I added "Camp Horno Chapel" to google maps using the add missing location button. Some or these different attempts will work on our site but none get added to the app.

I appreciate any help you can give.  I can be reached at cmzncaa@gmail.com or [phone deleted].
 

I have added several files below.
This is a list of our meeting on Camp Pendleton in both list and map mode:
<img width="1298" alt="Camp Pendleton Meeting List" src="https://github.com/code4recovery/12-step-meeting-list/assets/77302343/4ed5dbdf-700b-4f65-9561-43820f427c17">
<img width="938" alt="Camp Pendleton Meetings Map" src="https://github.com/code4recovery/12-step-meeting-list/assets/77302343/9751e7fc-ded7-4b31-8c4d-a8331d9b1a2a">
The following are the meeting displays and directions display from our site.
<img width="1106" alt="San Mateo Page" src="https://github.com/code4recovery/12-step-meeting-list/assets/77302343/8491c228-1b75-4c85-b03b-78a04b0d3aa4">
<img width="402" alt="San Mateo Directions" src="https://github.com/code4recovery/12-step-meeting-list/assets/77302343/5a30f141-c3a4-4093-98d7-4cadcc0c62cb">
<img width="635" alt="Camp Horno Directions" src="https://github.com/code4recovery/12-step-meeting-list/assets/77302343/fb60daf2-f5b5-4490-8e6c-3fec28afcbaa">
<img width="1215" alt="Camp Horno Page" src="https://github.com/code4recovery/12-step-meeting-list/assets/77302343/3f8bbb52-b1a6-4fde-80c5-f1a9665c71dc">
I have also included the csv for the two problem meetings.
[Camp Pendleton Problem Meetings.csv](https://github.com/code4recovery/12-step-meeting-list/files/15084142/Camp.Pendleton.Problem.Meetings.csv)

**Comment by ginslo** [comment]
Thanks for your detailed report. For issues related to the Meeting Guide app, the best path forward is to send a report to appsupport@aa.org as Code For Recovery does not manage the app. 

Actual addresses are always best, but as you explained, in the case of Camp Pendleton, that is not possible. The meeting with the + code, Camp Horno Discussion, should be okay actually but the  San Mateo Sober Nooners meeting is too approximate; needs additional detail. 

So again, best suggestion at this point is to submit a report to appsupport@aa.org. Please keep us posted here on the progress.

**Reply by tim-rohrer** [reply]
Hey @dev4a8, let me add a little bit to Gary's answer. 

First, to confirm, everything looks good on your website, correct? 

For the meetings that are missing in Meeting Guide, when  you contact App Support, you'll want to ask them to check the geofence boundaries. They can adjust them to make sure the meetings are accepted.

However, if the meetings are not "showing up" because the address you're getting to work in TSML doesn't work in Meeting Guide (which I admit is confusing because both systems use Google for geocoding), then you'll want to work with App Support to determine the most effective address overrides. 

HTH,

Tim

p.s. I took the liberty of redacting your phone number as this is a publicly visible system.
